src/du/du.rs

```diff
@@ -398,7 +422,9 @@ Try '{} --help' for more information.",
                     }
                 }
             }
-            Err(error) => show_error!("{}", error),
+            Err(_) => {
+                show_info!("{}: {}", path_str, "No such file or directory");
```

Comment:
These seems like it should be reported as an error, so I think `show_error!` is the correct macro to use here.

Reply:
I considered that. Yes it does seem to be an error but if we call _show_error!_ then the string 'Error' appears in the middle of the output. This makes the output different to 'C' du which does not have this. 

Quite happy to change this, just letting you know the error message will then be different to 'C' du.

Reply:
I don't think the GNU utils ever print `error: blah`, so basically everything that uses `show_error!` isn't 100% the same as the GNU version.  Maybe we should eventually add a `GNULY_COMPLIANT` variable or something lol.